doc/configcommands-linked.asciidoc

```diff
@@ -0,0 +1,1624 @@
+<<define-filter>> +
+<<delete-read-articles-on-quit>> +
+<<dialogs-title-format>> +
+<<dirbrowser-title-format>> +
+<<display-article-progress>> +
+<<download-full-page>> +
+<<download-retries>> +
+<<download-timeout>> +
+<<error-log>> +
+<<external-url-viewer>> +
+<<feed-sort-order>> +
+<<feedhq-flag-share>> +
+<<feedhq-flag-star>> +
+<<feedhq-login>> +
+<<feedhq-min-items>> +
+<<feedhq-password>> +
+<<feedhq-passwordfile>> +
+<<feedhq-passwordeval>> +
+<<feedhq-show-special-feeds>> +
+<<feedhq-url>> +
+<<feedlist-format>> +
+<<feedlist-title-format>> +
+<<filebrowser-title-format>> +
+<<goto-first-unread>> +
+<<goto-next-feed>> +
+<<help-title-format>> +
+<<highlight>> +
+<<highlight-article>> +
+<<history-limit>> +
+<<html-renderer>> +
+<<http-auth-method>> +
+<<ignore-article>> +
+<<ignore-mode>> +
+<<include>> +
+<<itemview-title-format>> +
+<<inoreader-flag-share>> +
+<<inoreader-flag-star>> +
+<<inoreader-login>> +
+<<inoreader-min-items>> +
+<<inoreader-password>> +
+<<inoreader-passwordfile>> +
+<<inoreader-passwordeval>> +
+<<inoreader-show-special-feeds>> +
+<<keep-articles-days>> +
+<<macro>> +
+<<mark-as-read-on-hover>> +
+<<max-download-speed>> +
+<<max-browser-tabs>> +
+<<max-items>> +
+<<newsblur-login>> +
+<<newsblur-min-items>> +
+<<newsblur-password>> +
+<<newsblur-passwordfile>> +
+<<newsblur-passwordeval>> +
+<<newsblur-url>> +
+<<notify-always>> +
+<<notify-beep>> +
+<<notify-format>> +
+<<notify-program>> +
+<<notify-screen>> +
+<<notify-xterm>> +
+<<ocnews-flag-star>> +
+<<ocnews-login>> +
+<<ocnews-password>> +
+<<ocnews-passwordfile>> +
+<<ocnews-passwordeval>> +
+<<ocnews-url>> +
+<<oldreader-flag-share>> +
+<<oldreader-flag-star>> +
+<<oldreader-login>> +
+<<oldreader-min-items>> +
+<<oldreader-password>> +
+<<oldreader-passwordfile>> +
+<<oldreader-passwordeval>> +
+<<oldreader-show-special-feeds>> +
+<<openbrowser-and-mark-jumps-to-next-unread>> +
+<<opml-url>> +
+<<pager>> +
+<<podcast-auto-enqueue>> +
+<<prepopulate-query-feeds>> +
+<<ssl-verifyhost>> +
+<<ssl-verifypeer>> +
+<<proxy-auth-method>> +
+<<proxy-auth>> +
+<<proxy-type>> +
+<<proxy>> +
+<<refresh-on-startup>> +
+<<reload-only-visible-feeds>> +
+<<reload-threads>> +
+<<reload-time>> +
+<<reset-unread-on-update>> +
+<<save-path>> +
+<<search-highlight-colors>> +
+<<searchresult-title-format>> +
+<<selectfilter-title-format>> +
+<<selecttag-format>> +
+<<selecttag-title-format>> +
+<<show-keymap-hint>> +
+<<show-title-bar>> +
+<<show-read-articles>> +
+<<show-read-feeds>> +
+<<suppress-first-reload>> +
+<<swap-title-and-hints>> +
+<<text-width>> +
+<<toggleitemread-jumps-to-next-unread>> +
+<<ttrss-flag-publish>> +
+<<ttrss-flag-star>> +
+<<ttrss-login>> +
+<<ttrss-mode>> +
+<<ttrss-password>> +
+<<ttrss-passwordfile>> +
+<<ttrss-passwordeval>> +
+<<ttrss-url>> +
+<<unbind-key>> +
+<<urls-source>> +
+<<urlview-title-format>> +
+<<use-proxy>> +
+<<user-agent>> +
+
+
+
+'''
+****
+[[always-display-description]]*Syntax:* always-display-description [yes/no] +
```

Comment:
I imagined that each item will link to itself, something like:

```
[[always-display-description]]*Syntax*: <<always-display-description>> [yes/no]
```

This way I can find the setting I want, click its link, and copy-paste that into a chat or whatever. With current layout, I have to go hunt for the link above the list, which is inconvenient.
```